SpatialGDK/Source/SpatialGDK/Public/Schema/StandardLibrary.h

```diff
@@ -177,6 +177,45 @@ struct Persistence : AbstractMutableComponent
 	}
 };
 
+struct SnapshotVersion : AbstractMutableComponent
```

Comment:
StandardLibrary is used for the standard library schema. This is schema definited in the global_state_manager.schema, so should go in it's own file. Could be grouped with other gsm c data definitions, but I'm not sure we have any.